app/models/concerns/thredded/post_common.rb

```diff
@@ -13,14 +13,18 @@ module PostCommon
 
       validates :content, presence: true
 
-      scope :order_oldest_first, -> { order(id: :asc) }
-      scope :order_newest_first, -> { order(id: :desc) }
+      scope :order_oldest_first, -> { order(created_at: :asc) }
```

Comment:
I'm not sure this is really covered by the birthday problem, but I'm not going to argue that one! Don't think #352 is really the same, but in general think the answer for test problems is to improve the test data! 

however if you've seen this in production, and for the sake of a more complex index and a few extra characters of code... I give in! 😃